arcgis-ios-sdk-samples/Content Display Logic/Controllers/AppInfoViewController.swift

```diff
@@ -14,40 +14,167 @@
 // limitations under the License.
 
 import UIKit
+import ArcGIS
 
-class AppInfoViewController: UIViewController {
-    @IBOutlet var appNameLabel: UILabel!
-    @IBOutlet var appVersionLabel: UILabel!
+class AppInfoViewController: UITableViewController {
+    private let links = [
+        Link(title: "Esri Community", url: URLs.esriCommunity),
+        Link(title: "GitHub Repository", url: URLs.githubRepository),
+        Link(title: "Sample Code", url: URLs.sampleCode)
```

Comment:
This is a good idea, but I don't know where to put these text. 
- Do you think putting them as caption of the table cell makes sense?
- The other idea is to put them in the footer of the section, similar to the copyright text

Reply:
What if each link were its own section so you can add them as headers?

Reply:
Section headers won't work because they have a default CAPITALIZED font, unless we override the header views in whole. I read the [table view HIG](https://developer.apple.com/design/human-interface-guidelines/ios/views/tables/) but didn't get much inspiration. Maybe it's time to design a custom cell?

Reply:
Ohh I see. I like this idea!

Reply:
Options

1. Put it as the subtitle of a subtitle cell

![subtitle](https://user-images.githubusercontent.com/9660181/146849375-7053fee7-19e8-4109-8867-df0ef71de4d0.png)

2. Mimic the system settings app, e.g. General -> iPhone Storage, or like this, to put more info as section footer

![Simulator Screen Shot - iPhone 11 - 2021-12-17 at 16 59 47](https://user-images.githubusercontent.com/9660181/146623633-43098511-8600-40c3-ad54-410794567337.png)

3. Custom cell

Reply:
I think option 2 looks great!

Reply:
Yup, maybe it worth rewriting this PR in SwiftUI. 🚀  I'll change this to draft and set up a different PR with SwiftUI.

Reply:
Please see #1132 for the SwiftUI design and share your thoughts!